appholder/src/main/java/com/android/identity/wallet/support/androidkeystore/AndroidAuthKeyCurveOption.kt

```diff
@@ -1,18 +1,19 @@
 package com.android.identity.wallet.support.androidkeystore
 
 import android.os.Parcelable
+import com.android.identity.securearea.EcCurve
 import com.android.identity.securearea.SecureArea
 import kotlinx.parcelize.Parcelize
 
 @Parcelize
 enum class AndroidAuthKeyCurveOption : Parcelable {
     P_256, Ed25519, X25519;
 
-    fun toEcCurve(): Int {
+    fun toEcCurve(): EcCurve {
         return when (this) {
-            P_256 -> SecureArea.EC_CURVE_P256
-            Ed25519 -> SecureArea.EC_CURVE_ED25519
-            X25519 -> SecureArea.EC_CURVE_X25519
+            P_256 -> EcCurve.P256
+            Ed25519 -> EcCurve.ED25519
+            X25519 -> EcCurve.X25519
```

Comment:
I know your changes are about using the enum `EcCurve` instead of Ints, but it touches code that could also be refactored to be more Kotlin DSL 

```
    fun toEcCurve() = when (this) {
        P_256 -> EcCurve.P256
        Ed25519 -> EcCurve.ED25519
        X25519 -> EcCurve.X25519
    }
```